tests/performance/results.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const testSuites = [
+    'home-classic-theme',
+    'home-block-theme',
+];
+
+console.log( '\n>> 🎉 Results 🎉 \n' );
+
+function median( array ) {
+	const mid = Math.floor( array.length / 2 ),
+		numbers = [ ...array ].sort( ( a, b ) => a - b );
```

Comment:
Nitpick for readability, even though I know this is a direct copy of the same function from Gutenberg's perf tests.
```suggestion
	const mid = Math.floor( array.length / 2 );
	const numbers = [ ...array ].sort( ( a, b ) => a - b );
	
```

Reply:
Updated in https://github.com/10up/wordpress-develop/pull/16/commits/7aa7a3c2bf490be7c09a240c3453e052a5959012 Thanks.